doc/developer-notes.md

```diff
@@ -606,7 +606,7 @@ To create a scripted-diff:
 
 The scripted-diff is verified by the tool `contrib/devtools/commit-script-check.sh`
 
-Commit `bb81e173` is an example of a scripted-diff.
+Commit [`bb81e173`](https://github.com/bitcoin/bitcoin/commit/bb81e173) is an example of a scripted-diff.
```

Comment:
I think if you remove the back ticks it will link.

Reply:
@promag I think github is clever in that regard, but I still see added value - if you only have a copy of that file and view it in vim/emacs/notepad or a plain markdown renderer.